app/presenters/curation_concerns/inspect_work_presenter.rb

```diff
@@ -0,0 +1,66 @@
+module CurationConcerns
+  class InspectWorkPresenter
+    def initialize(solr_document, current_ability)
+      @solr_document = solr_document
+      @current_ability = current_ability
+    end
+    attr_reader :solr_document, :current_ability
+
+    def workflow
+      workflow_resp = { comments: [], roles: [] }
+      if sipity_entity
```

Comment:
Use a guard clause here instead.  This means `return workflow_resp if sipity_entity`  then the next line you can just define the Hash (no need to assign it to `workflow_resp`